elsie/slides/slidedeck.py

```diff
@@ -405,25 +327,29 @@ def render(
             slide_postprocessing([slide.box() for slide in select_slides])
 
         if prune_cache:
-            self.query_cache = self.used_query_cache
-            self.used_query_cache = {}
+            self.backend.prune_cache()
 
         if save_cache:
-            self._save_query_cache(self.query_cache)
```

Comment:
query cache is somehow independent on the backend, more precisely it stays over the backends, so saving the case should not probably be part of the backend. But I will have to think about it more.